syncdata-default.properties

```diff
@@ -343,6 +343,9 @@ datastores=ldap_1_DS,db_1_DS,db_2_DS
 ## Partner Management Service allowed partner domains
 mosip.kernel.partner.allowed.domains=AUTH,DEVICE,FTM
 
+## List of keys to auto generate.
+mosip.kernel.keymanager.autogen.appids.list=ROOT,KERNEL:SIGN,PRE_REGISTRATION,REGISTRATION,REGISTRATION_PROCESSOR,ID_REPO,KERNEL:IDENTITY_CACHE,RESIDENT,PMS
```

Comment:
This is not required in syncdata-default.properties

Reply:
removed